stonesoup/moveable/movable.py

```diff
@@ -0,0 +1,532 @@
+        towards the origin.
+
+        The orientation of this platform is defined as along the direction of its velocity, with
+        roll always set to zero (as this is the angle the platform is rotated about the velocity
+        axis, which is not defined in this approximation).
+
+        Notes
+        -----
+        A non-moving platform (``self.is_moving == False``) does not have a defined orientation in
+        this approximations and so raises an :class:`AttributeError`
+        """
+        if not self.is_moving:
+            raise AttributeError('Orientation of a zero-velocity moving platform is not defined')
+        velocity = self.velocity
+
+        if self.ndim == 3:
+            _, bearing, elevation = cart2sphere(*velocity.flat)
+            return StateVector([0, elevation, bearing])
+        elif self.ndim == 2:
+            _, bearing = cart2pol(*velocity.flat)
+            return StateVector([0, 0, bearing])
+        else:
+            raise NotImplementedError('Orientation of a moving platform is only implemented for 2'
+                                      'and 3 dimensions')
+
+    @property
+    def is_moving(self) -> bool:
+        """Return the ``True`` if the platform is moving, ``False`` otherwise.
+
+        Equivalent (for this class) to ``all(v == 0 for v in self.velocity)``
+        """
+        # Note: a platform without a transition model can be given a velocity as part of it's
+        # StateVector. It just won't move
+        # This inconsistency is handled in the move logic
+        return np.any(self.velocity != 0)
+
+    def _set_position(self, value: StateVector):
+        # The logic below is this: if a moving platform is being built from (say) input
+        # real-world data then its transition model would not be set, and so it would be fine to
+        # set its position. However, if the transition model is set, then setting the position is
+        # both unexpected and may cause odd effects, so is forbidden
+        if self.transition_model is None:
+            self.state_vector[self.position_mapping, :] = value
+        else:
+            raise AttributeError('Cannot set the position of a moving platform with a '
+                                 'transition model')
+
+    def move(self, timestamp=None, **kwargs) -> None:
+        """Propagate the platform position using the :attr:`transition_model`.
+
+        Parameters
+        ----------
+        timestamp: :class:`datetime.datetime`, optional
+            A timestamp signifying when the end of the maneuver \
+            (the default is ``None``)
+
+        Notes
+        -----
+        This methods updates the value of :attr:`position`.
+
+        Any provided ``kwargs`` are forwarded to the :attr:`transition_model`.
+
+        If :attr:`transition_model` or ``timestamp`` is ``None``, the method has
+        no effect, but will return successfully.
```

Comment:
This is false in the instance there is no transition model. An `AttributeError` is raised.

Reply:
Documentation is correct to match the behaviour of the code.